planner/core/plan_to_pb_test.go

```diff
@@ -79,4 +79,19 @@ func TestColumnToProto(t *testing.T) {
 	}
 	pc = util.ColumnToProto(col2, false)
 	require.Len(t, pc.Elems, 2)
+
+	tp = types.NewFieldTypeBuilder().
+		SetType(mysql.TypeString).
+		SetCharset("utf8mb4").
```

Comment:
If we'll modify the collation to 'binary', why setting 'utf8mb4'/'utf8mb4_bin' in this case?

Reply:
The `tp` here is the input of `util.ColumnToProto`. This test shows the function `util.ColumnToProto` can converts any collation (if `.array` is true) into `binary` collation. The expected output is defined in L94:

```
expect = &tipb.ColumnInfo{ColumnId: 0, Tp: 0xfe, Collation: 63, ColumnLen: 100, Decimal: 0, Flag: 10, Elems: []string(nil), DefaultVal: []uint8(nil), PkHandle: false, XXX_unrecognized: []uint8(nil)}
```

The `Collation: 63` means binary collation.